src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +15,13 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+
+    public static User findByEmail(String email) {
```

Comment:
Don’t use static methods in your solution
Static methods are in general a bad practice. Let’s better create an instance of a class which method you want to call.
```suggestion
    public User findByEmail(String email) {
```